statsmodels/regression/quantile_regression.py

```diff
@@ -116,7 +116,7 @@ def fit(self, q=.5, vcov='robust', kernel='epa', bandwidth='hsheather',
             - chamberlain: Chamberlain (1994)
         """
 
-        if q < 0 or q > 1:
+        if q <= 0 or q >= 1:
             raise Exception('p must be between 0 and 1')
```

Comment:
Could you replace between with strictly between?

Reply:
Fixed in 9c0f06e. I also adjusted the docstring.